tests/conftest.py

```diff
@@ -43,12 +43,13 @@ def metadata(_mock_timestamp: None) -> DataDocMetadata:
     return DataDocMetadata(str(TEST_PARQUET_FILEPATH))
 
 
-@pytest.fixture()
+@pytest.fixture(autouse=True)
 def remove_document_file() -> Generator[None, None, None]:
     # Yield so we only run teardown
     yield None
     try:
-        TEST_RESOURCES_METADATA_DOCUMENT.unlink()
+        if TEST_RESOURCES_METADATA_DOCUMENT.exists():
```

Comment:
Is this check necessary? We should have either this check or the try/except, not both, you can decide which to remove 😄